src/python/dxpy/templating/templates/nextflow/src/nextflow.sh

```diff
@@ -454,14 +454,15 @@ get_nextflow_environment() {
 
 log_context_info() {
   echo "============================================================="
-  echo "=== NF projectDir   : @@RESOURCES_SUBPATH@@"
-  echo "=== NF session ID   : ${NXF_UUID}"
-  echo "=== NF log file     : dx://${DX_JOB_OUTDIR%/}/${LOG_NAME}"
+  echo "=== NF command          :" "${NEXTFLOW_CMD[@]}"
+  echo "=== NF session ID       : ${NXF_UUID}"
+  echo "=== NF workdir          : ${NXF_WORK}"
   if [[ $preserve_cache == true ]]; then
-    echo "=== NF cache folder : dx://${DX_CACHEDIR}/${NXF_UUID}/"
```

Comment:
This is now covered by printing "NF workdir..." always

Reply:
Would this be a breaking change to current users? For example, what if users rely on reading/parsing the existing log format? 
What is the down side of keeping the existing log but adding new ones?

Reply:
Please address the command. Once it is resolved, this LGTM. Thanks!

Reply:
I reverted the other changes besides adding the "=== NF workdir      : ${NXF_WORK}" line.